### PR TITLE
Updated README.md and deployment.md for Azure.

### PR DESCRIPTION
**Pull request by gingerhelp**
Pull request to add deployment documentation for Azure.

### CODE REVIEW

**Review by vineyardbovines — APPROVED** [review]

**Comment by vineyardbovines** [comment]
Thanks @gingerhelp!

File: docs/general/deployment.md [hunk 1]
```diff
@@ -119,3 +119,17 @@ _Step 4:_ commit your changes and deploy via EB CLI:
...
+### Easy 3-Step Deployment Process
+_Step 1:_ Within Azure Portal, add a 'Web App' resource to your resource group.  The runtime stack must be one of the Node versions.  Please note that much of the Microsoft quick starts out result in Node being run via IIS which will not work with react-boilerplate.
+
+_Step 2:_ When the resource has finished deploying, go to it's deployment center and select Local Git (other methods will work as well but the rest of these steps assume this approach) and 'App Service' for the build provider.  Note the Git Clone Uri that is presented when the wizard is finished.
+
+_Step 3:_ Within the root of your react-boilerplate source folder, execute the following commands to publish to Azure:
+
+1.  `git remote add azure https://YOUR_RESOURCE_NAME.scm.azurewebsites.net:443/YOUR_RESOURCE_NAME.git`
+2.  `git add .`
+3.  `git commit -m 'Made some epic changes as per usual'`
```
Comment (julienben) [inline docs/general/deployment.md:134]: 🤣

File: docs/general/deployment.md [hunk 2]
```diff
@@ -119,3 +119,17 @@ _Step 4:_ commit your changes and deploy via EB CLI:
 ```sh
 eb deploy {target environment name}
 ```
+
+## Azure
+
+### Easy 3-Step Deployment Process
+_Step 1:_ Within Azure Portal, add a 'Web App' resource to your resource group.  The runtime stack must be one of the Node versions.  Please note that much of the Microsoft quick starts out result in Node being run via IIS which will not work with react-boilerplate.
+
+_Step 2:_ When the resource has finished deploying, go to it's deployment center and select Local Git (other methods will work as well but the rest of these steps assume this approach) and 'App Service' for the build provider.  Note the Git Clone Uri that is presented when the wizard is finished.
```
Comment (julienben) [inline docs/general/deployment.md]: Can you fix a spelling mistake here? "it's" should be "its".

File: docs/general/deployment.md [hunk 3]
```diff
@@ -119,3 +119,17 @@ _Step 4:_ commit your changes and deploy via EB CLI:
 ```sh
 eb deploy {target environment name}
 ```
+
+## Azure
+
+### Easy 3-Step Deployment Process
+_Step 1:_ Within Azure Portal, add a 'Web App' resource to your resource group.  The runtime stack must be one of the Node versions.  Please note that much of the Microsoft quick starts out result in Node being run via IIS which will not work with react-boilerplate.
```
Comment (julienben) [inline docs/general/deployment.md]: I'm slightly confused here. Is there an additional step necessary for making sure Node isn't being run via IIS?

**Review by julienben — CHANGES_REQUESTED** [review]
Thanks for your contribution @gingerhelp! 

Please see a couple of small comments.

File: docs/general/deployment.md (hunk 2 above)
Comment (gingerhelp) [reply docs/general/deployment.md]: Yup - just fixed that!

File: docs/general/deployment.md (hunk 3 above)
Comment (gingerhelp) [reply docs/general/deployment.md]: Thanks @julienben - I have added a little extra description to the step to hopefully make it a little more clear.  Let me know if it makes more sense now.

**Review by julienben — APPROVED** [review]

**Comment by julienben** [comment]
Thanks for the fix and for your contribution!!

**Pull request merged**